src/V4/PeanutV4.sol

```diff
@@ -419,6 +483,15 @@ contract PeanutV4 is IERC721Receiver, IERC1155Receiver, ReentrancyGuard {
         return true;
     }
 
+    function withdrawDepositSender(uint256 _index) external nonReentrant returns (bool) {
+        return _withdrawDepositSender(_index, msg.sender);
+    }
+
+    function withdrawDepositSenderGasless(GaslessReclaim calldata reclaim, address signer, bytes calldata signature) external nonReentrant returns (bool) {
+        verifyGaslessReclaim(reclaim, signer, signature);
```

Comment:
Shouldn't this `require()` verifyGaslessReclaim? 

```suggestion
    require(verifyGaslessReclaim(reclaim, signer, signature), "INVALID SIGNATURE");
```

Reply:
Actually inside `verifyGaslessReclaim` there already is `require(valid, "INVALID SIGNATURE");`

Perhaps I could adjust `verifyGaslessReclaim` so that it doesn't return anything, so that it's more clear?

Reply:
Yes, if you're going to return a bool in verifyGasless, then that value should have some meaning (now it always returns true, or raises)